### PR TITLE
Arafat adiham

### PR DESCRIPTION
**Pull request by arafatadiham**

### CODE REVIEW

**Comment by arafatadiham** [comment]
Hi Jordan , can you please merge to master branch

File: src/test/java/com/automation/utilities/Driver.java [hunk 1]
```diff
@@ -21,11 +22,20 @@ public static WebDriver getDriver(){
             String browser = ConfigurationReader.getProperty("browser").toLowerCase();
             switch (browser){
                 case "chrome":
-                    WebDriverManager.chromedriver().version("79").setup();
-                    driver = new ChromeDriver();
+                    WebDriverManager.chromedriver().setup();
```
Comment (jlhankison) [inline src/test/java/com/automation/utilities/Driver.java]: Can you add .version("79") here?

File: src/test/java/com/automation/utilities/Driver.java [hunk 2]
```diff
@@ -21,11 +22,20 @@ public static WebDriver getDriver(){
...
+                case "chromeheadless":
+                    //to run chrome without interface (headless mode)
+                    WebDriverManager.chromedriver().setup();
+                    ChromeOptions options = new ChromeOptions();
+                    options.setHeadless(true);
+                    driver = new ChromeDriver(options);
                     break;
                 case "firefox":
-                    WebDriverManager.firefoxdriver().version("79").setup();
+                    WebDriverManager.firefoxdriver().setup();
```
Comment (jlhankison) [inline src/test/java/com/automation/utilities/Driver.java]: same here

**Review by jlhankison — CHANGES_REQUESTED** [review]
looks pretty good! Just check those changes and you should be set for the pull

**Pull request closed without merging**